src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/ReadyToRunMethod.cs

```diff
@@ -554,13 +556,20 @@ private void EnsureFixupCells()
         private void ParseRuntimeFunctions(bool partial)
         {
             int runtimeFunctionId = EntryPointRuntimeFunctionId;
+            int coldRuntimeFunctionId = ColdRuntimeFunctionId;
             int runtimeFunctionSize = _readyToRunReader.CalculateRuntimeFunctionSize();
             int runtimeFunctionOffset = _readyToRunReader.CompositeReader.GetOffset(_readyToRunReader.ReadyToRunHeader.Sections[ReadyToRunSectionType.RuntimeFunctions].RelativeVirtualAddress);
             int curOffset = runtimeFunctionOffset + runtimeFunctionId * runtimeFunctionSize;
+            int coldOffset = runtimeFunctionOffset + coldRuntimeFunctionId * runtimeFunctionSize;
             int codeOffset = 0;
 
             for (int i = 0; i < RuntimeFunctionCount; i++)
             {
+                if (i == (RuntimeFunctionCount - ColdRuntimeFunctionCount))
```

Comment:
It might make the code slightly more readable and performant at the same time to calculate this difference upfront into a new local variable <code>hotRuntimeFunctionCount</code> if I'm right to understand that's what the difference represents.